Sources/AblyAssetTrackingInternal/WorkerQueue/WorkerQueueProperties.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+
+/// A protocol that contains prperties that affect a ``WorkerQueue``'s behaviour.
+public protocol WorkerQueueProperties {
+    var isStopped: Bool { get set }
```

Comment:
b92be3af7e9f4fb4bc0e8bb882d2a7bfe059b154

Reply:
A couple of things:

- Can we document how `isStopped` gets set? Is this something that a worker can set if they wish to stop the queue, or is it something that the queue itself sets? (I believe it's the former?)
- > If a new worker gets added to a stopped Worker Queue, the `Worker.doWhenStopped` is executed instead.

  I don't think it's just new workers that this applies to, is it? Isn't it that, after a worker returns properties with `isStopped == true`, _all_ subsequent workers (including those already queued) will have their `doWhenStopped` executed?

Reply:
Hmm, in Android it's not always set by workers - in [PublisherProperties](https://github.com/ably/ably-asset-tracking-android/blob/6b09c9ca180f8958c69e41c9c1f2514941eaad14/publishing-sdk/src/main/java/com/ably/tracking/publisher/PublisherProperties.kt#L135) it's actually tied to the Publisher's state. In Subscriber I see that it can actually be changed by workers.

Dunno if we will want to follow the Publisher's approach from Android without digging in too deep, so I'd personally not specify this part right now.
I did update the docstring according to your first point though: c40208564a1c09a01905e662593e122bbf1e5026

Reply:
Cool, thanks 🙂